### PR TITLE
x11-misc/ftmenu: EAPI bump 2 -> 6

### PR DESCRIPTION
**Pull request by Moiman**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

File: x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild [hunk 1]
```diff
@@ -20,15 +20,6 @@ DEPEND="${COMMON_DEPEND}
 RDEPEND="${COMMON_DEPEND}
 	x11-wm/fluxbox"
 
-src_compile() {
-	emake CFLAGS="${CFLAGS}" || die "emake failed"
```
Comment (SoapGentoo) [inline x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild]: have you checked that `CFLAGS` are respected?

File: x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild [hunk 2]
```diff
@@ -20,15 +20,6 @@ DEPEND="${COMMON_DEPEND}
...
-}
-
-src_install() {
-	make DESTDIR="${D}" install || die "make install failed"
-	dodoc AUTHORS NEWS README ChangeLog || die
-}
-
 pkg_postinst() {
 	echo
```
Comment (SoapGentoo) [inline x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild]: `echo` -> `einfo`

File: x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild (hunk 1 above)
Comment (Moiman) [inline x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild]: yes
```
>>> Compiling source in /var/tmp/portage/x11-misc/ftmenu-0.4.0-r1/work/ftmenu-0.4.0 ...
make -j2 
Making all in src
make[1]: Entering directory '/var/tmp/portage/x11-misc/ftmenu-0.4.0-r1/work/ftmenu-0.4.0/src'
x86_64-pc-linux-gnu-gcc -DPACKAGE_NAME=\"\" -DPACKAGE_TARNAME=\"\" -DPACKAGE_VERSION=\"\" -DPACKAGE_STRING=\"\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DPACKAGE=\"ftmenu\" -DVERSION=\"0.4.0\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -I. -pthread -I/usr/include/gtk-2.0 -I/usr/lib64/gtk-2.0/include -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/freetype2    -march=k8-sse3 -O2 -pipe -ggdb -c ftmenu.c
/bin/sh ../libtool --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -march=k8-sse3 -O2 -pipe -ggdb  -Wl,-O1 -Wl,--as-needed -o ftmenu ftmenu.o -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype  
libtool: link: x86_64-pc-linux-gnu-gcc -march=k8-sse3 -O2 -pipe -ggdb -Wl,-O1 -Wl,--as-needed -o ftmenu ftmenu.o  -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype
make[1]: Leaving directory '/var/tmp/portage/x11-misc/ftmenu-0.4.0-r1/work/ftmenu-0.4.0/src'
make[1]: Entering directory '/var/tmp/portage/x11-misc/ftmenu-0.4.0-r1/work/ftmenu-0.4.0'
make[1]: Nothing to be done for 'all-am'.
make[1]: Leaving directory '/var/tmp/portage/x11-misc/ftmenu-0.4.0-r1/work/ftmenu-0.4.0'
>>> Source compiled.
```

Comment (SoapGentoo) [inline x11-misc/ftmenu/ftmenu-0.4.0-r1.ebuild]: thanks for checking!

**Pull request closed without merging**